### ISSUE TITLE
Show with CLI command?

### ISSUE THREAD

**Issue by saulfautley** [Ulauncher/Ulauncher#415]
## Question
Is it possible show ULauncher with a cli command?

There seems to be quite a few limitations for the `Hotkey` that can be used, which makes sense (e.g. using `super` as a modifier or on its own). But being able to open ULauncher with a command would allow setting a hotkey at the system level, as well as possibly enabling other cool ways to use the launcher, like with mouse buttons, gestures or even custom voice prompts.

I know you can run `ulauncher`, but a) it has a slight delay compared to using the builtin hotkey, and b) it doesn't focus the input box automatically.

**Comment by arigit** [comment]
FWIW, I use Super+Alt+Space with not issue as hotkey with ulauncher
In my office macbook I configured spotlight to use the same, so both mac and linux desktops are aligned on the same shortcut.
Also, there is a way to change gnome-shell's overivew hotkey from Super to something else which would allow you to use Super+Space in ulauncher, but I found that that breaks gnome-shell experience too much. It was simpler to just change spotlight, if your goal is to have the same shortcut in both linux and mac.
